### ISSUE TITLE
PPSSPP crashing when tapping more than once

### ISSUE THREAD

**Issue by Kahsai** [hrydgard/ppsspp#8351]
Hey whenever I try and use ppsspp I have noticed that if I tap more than one time it will crash the application and I don't know if you guys already know about this issue. I don't even have to be in a game for this to happen. I am using an iPhone 6s running iOS 9.0.2. Also I have you guys fixed the issue that causes games to crash before they have fully started? Because the only game I have been able to enter is naruto ultimate ninja heroes but I can't actually fight because when I tap more than once it will crash the game but I do successfully get into the vs match without any issue. Any other game like God of war or bleach will crash before I actually enter the game. Anyways please help fix this issue as soon as you can. Thanks.

**Comment by unknownbrackets** [comment]
Are you using the latest git build?  Not v1.1.1 or anything.

Be aware that if you're running iOS 9, you should expect crashing - see #8122.

-[Unknown]

**Comment by Kahsai** [comment]
I am using the one that says Dev-latest. It is v1.1.1-532-g337f27d. And yes I know that it is currently not working for iOS 9 but I am able to get into naruto ultimate ninja heroes and get into a match but it crashes when I try and fight because I have to tap buttons multiple times. But it isn't just in a game. Let's say I just open the ppsspp app and have not selected a game yet. If I double tap the screen ppsspp will crash. I think this is a desperate bug from the issue of it not working on iOS 9. Sorry if it's kind of hard to understand what I am writing I am rushed to type this.

**Comment by unknownbrackets** [comment]
Well, I just asked someone else on 9.0.2 to test and it doesn't crash for them, tapping with multiple fingers, one, repeatedly, double tapping, etc.  No crashes.  In the UI and in a game.

-[Unknown]

**Comment by Kahsai** [comment]
No I close out of everything. Are they running the same version of ppsspp as me? Also I meant tapping repeatedly with one finger really fast. If I take one finger and tap twice back to back really fast it crashes ppsspp

**Comment by Kahsai** [comment]
Hmm actually I have a theory on what might be causing it I'll let you know if I am right

**Comment by Kahsai** [comment]
Yea I was right I'm using a tweak called videopane. It's so that I can double tap a video and do picture in picture but seems like it was accidentally trying to do the same thing with ppsspp as well which was resulting in a crash. Sorry for wasting your time lol. But now that I disabled that I was able to successfully run naruto ultimate ninja heroes without crashing. I can fight no problems. But God of war and bleach are crashing for me? Any ideas on why one game will work but the other won't?
